src/Schema/DirectiveRegistry.php

```diff
@@ -130,157 +156,119 @@ public function get($name)
         // Always return a new instance of the directive class to avoid side effects between them
         return resolve(\get_class($directive));
     }
-
-    /**
-     * Get directive instance by name.
-     *
-     * @param string $name
-     *
-     * @throws DirectiveException
-     *
-     * @return Directive
-     *
-     * @deprecated Will be removed in next major release
-     */
-    public function handler($name)
-    {
-        return $this->get($name);
-    }
-
+    
     /**
-     * Get all directives associated with a node.
+     * Get all directives of a certain type that are associated with an AST node.
      *
      * @param Node $node
+     * @param string $directiveClass
      *
-     * @return \Illuminate\Support\Collection
+     * @return Collection
      */
-    protected function directives(Node $node)
+    protected function associatedDirectivesOfType(Node $node, string $directiveClass): Collection
```

Comment:
Question, what does the "associated" in the method name mean?
was it meant for this line `$this->hydrate($directive, $node);`?

Reply:
I caught myself adding a comment to the PHPDoc that explains how this gets just the directives that are associated with the specific node that is passed into this function. As a general rule of thumb, i try to avoid such comments so i just made the method name more explicit.

Reply:
I think `directivesOfType` is clear enough. It's confusing with the "associated" added, because user have to think "associate to what ?" for that name. 
At least, I got confused💦.